docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,364 @@
+---
+title: Host Certificates with Machine ID
+description: Issuing Host Certificates with Machine ID for OpenSSH servers.
+---
+
+In this guide, you will learn how to create host certificates through Machine ID to allow Teleport users (manually or automated through Machine ID) to access OpenSSH servers. This allows for the benefits of
+Machine ID with OpenSSH nodes and reduces risk by ensuring that short-lived host certificates on Teleport clusters can be successfully applied.
```

Comment:
I think that this doesn't really communicate precisely the benefit of using Machine ID rather than manually producing the certificate with `tctl`. I'd probably suggest that it reduces risk because you do not need to issue long-lived host certificates, which can be stolen and used by a malicious actor until they expire, and reduces the burden of needing to manually rotate these certificates.

Reply:
This is a good point, I'm expanding on this section to get the details out of the way at the start